src/app/services/selector_matcher.test.js

```diff
@@ -442,19 +442,19 @@ test("Test getting nodes by test type", () => {
     expect(
         matchByTestType('data')
     ).toStrictEqual(
-        [7]
+        []
```

Comment:
@leahwicz could you (or someone from the Core team) confirm that the changes to these tests are correct? I _think_ that these test were invalid and, if I'm reading this correctly, the node with id=7 is a generic/schema, not a singular/data test.

Reply:
@emmyoop could you be a second pair of eyes here please?